shell/platform/windows/platform_handler.cc

```diff
@@ -1,62 +1,72 @@
-// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Why are these diffs so weird?  Is my editor using the wrong newline character or something?